README.md

```diff
@@ -1,16 +1,32 @@
+## Pre-requisites
+
+- You will need to have an Azure account already available
+- Azure CLI installed to use this provider.
+- An Azure OpenAI endpoint provisioned with the models you would like to use.
```

Comment:
These feel kind of silly and obvious aside from the Azure CLI line.. do we really need to say this? I assume people who want to use Azure will understand that they can't just fire requests into the void without setting up an account and stuff.

Azure itself makes it confusing, but unless we want to get in the business of documenting how to deploy Azure OpenAI, I don't see why we would mention this at all.

Reply:
I can see the first bullet being a bit redundant because of the third bullet. 
The third bullet point wasn't obvious to me. 
No body reads the manual :)

Reply:
> No body reads the manual :)

_cries forever_